CHANGELOG.md

```diff
@@ -138,6 +138,7 @@ Skeleton (<https://github.com/cvat-ai/cvat/pull/1>), (<https://github.com/opencv
 - Fix build dependencies for Siammask (<https://github.com/openvinotoolkit/cvat/pull/4486>)
 - Bug: Exif orientation information handled incorrectly (<https://github.com/openvinotoolkit/cvat/pull/4529>)
 - Fixed build of retinanet function image (<https://github.com/cvat-ai/cvat/pull/54>)
+- Dataset import for Datumaro format (<https://github.com/opencv/cvat/pull/4544>)
```

Comment:
If this is about the Datumaro format, why are there also changes to other formats (KITTI and VGGFace)?

Reply:
The issue is related to the Datumaro format. Updated Changelog.